src/manipulateToDo.js

```diff
@@ -45,20 +82,153 @@ const drop = (ev) => {
 
   if (prevNextIds.prevId < prevNextIds.nextId) {
     setTimeout(() => {
-      document.getElementById(id).style.backgroundColor = '#4c3c3c';
+      let elUnder = document.getElementById(id);
+      // document.getElementById(id).style.backgroundColor = "#4c3c3c";
+      const classArray = elUnder.classList.value.split(' ');
+      if (
+        classArray.includes('fa-check')
+        || classArray.includes('checkbox')
+        || classArray.includes('to-do')
+      ) {
+        elUnder = elUnder.parentElement.parentElement;
+      } else if (
+        classArray.includes('fa-arrows-alt')
+        || classArray.includes('two')
+      ) {
+        elUnder = elUnder.parentElement;
+      }
+
+      elUnder.style.backgroundColor = '#4c3c3c';
+      elUnder.style.color = '#fff';
+      Array.from(elUnder.childNodes).forEach((node) => {
+        if (node.nodeName === 'DIV') {
+          const nodes = node.childNodes;
+          Array.from(nodes).forEach((one) => {
+            one.style.backgroundColor = '#4c3c3c';
+            one.style.color = '#fff';
+          });
+        }
+        node.style.backgroundColor = '#4c3c3c';
+        node.style.color = '#fff';
+      });
       setTimeout(() => {
-        document.getElementById(id).style.backgroundColor = '#fff';
+        // document.getElementById(id).style.backgroundColor = "#fff";
+        let elUnder = document.getElementById(id);
+        // document.getElementById(id).style.backgroundColor = "#4c3c3c";
+        const classArray = elUnder.classList.value.split(' ');
+        if (
+          classArray.includes('fa-check')
+          || classArray.includes('checkbox')
+          || classArray.includes('to-do')
+        ) {
+          elUnder = elUnder.parentElement.parentElement;
+        } else if (
+          classArray.includes('fa-arrows-alt')
+          || classArray.includes('two')
+        ) {
+          elUnder = elUnder.parentElement;
+        }
```

Comment:
This same code is repeating through this file several times. Making it a separate function would improve readability and stick a bit better to the YAGNI, what do you think? 🤓